tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md

```diff
@@ -1,39 +1,227 @@
+Test 19-8 VIC Upgrade
 =======
 
 # Purpose:
-To verify VIC 1.2 upgrade from 1.1.0 or 1.1.1 works and provides access to all of the vSphere Integrated Containers components.
+To verify VIC upgrade works and provides access to all of the vSphere Integrated Containers components.
 
 # References:
-[VIC 1.2 Upgrade](TBD - waiting on official docs to link)
-
-# Environment:
-* This test requires that a vCenter server is running and available.
-* Existing vSphere Integrated Containers 1.1.x appliance that has deployed by using the official OVA installer.
-* Configure and install VCH
-* Deploy a variety of containers through command line and admiral
-
-# Test Steps:
-1. Deploy new 1.2 OVA of the vSphere Integrated Containers appliance to a location in which it can access the VMDK files of the previous appliance.
-2. Log in to a vSphere Web Client instance to access both versions of the vSphere Integrated Containers appliance.
-3. Shut down both the previous and new vSphere Integrated Containers appliances by selecting Shut Down Guest OS.
-4. Click on the previous vSphere Integrated Containers appliance, and Remove data disk from VM
-5. Click on the new vSphere Integrated Containers appliance.
-6. Remove and delete empty data disk from new OVA VM
-7. Attach old data disk to new OVA VM
-8. Power on the new appliance to complete the upgrade
-9. Run the upgrade script to migrate harbor and admiral
-10. Run the vic-machine upgrade command to upgrade your existing VCH to the latest version
-11. Verify that the data from your previous vSphere Integrated Containers installation has migrated successfully
-12. Verify that VCH and containers from previous VIC appliance exists after upgrade and run docker commands.
-13. Deploy a variety of containers through command line and admiral on new VIC appliance.
-14. Remove each of the containers created
-15. Delete VCH
-16. Delete new and old OVA VM
-
-
-# Expected Outcome:
+[Oficial OVA download](https://my.vmware.com/en/web/vmware/info/slug/datacenter_cloud_infrastructure/vmware_vsphere_integrated_containers/1_2)
+
+[GCP Releases](https://console.cloud.google.com/storage/browser/vic-product-ova-releases?project=eminent-nation-87317&authuser=0)
+
+[VIC Engine Releases](https://console.cloud.google.com/storage/browser/vic-engine-releases?project=eminent-nation-87317&authuser=0)
+
+[OVA Smoke Test](https://confluence.eng.vmware.com/pages/viewpage.action?pageId=242300713)
+
+# Common Environment Setup:
+* These tests requires that a vCenter server is running and available with DRS enabled.
+
+# Test Template
+### Test Steps
+1. Deploy **[Deploy Old OVA Link]** previous version **[OVA Old Version]** of OVA of the vSphere Integrated Containers appliance
+2. Go to URL http://<appliance_IP> in browser
+3. Login to management portal using the credentials set during OVA deployment
+4. Go to **[Root Cert Path]** and download root certificate file
+5. Get respective version of VIC engine binaries
```

Comment:
We should say here that the VIC Engine binaries must be downloaded from the link in the Getting Started page (`https://<appliance_IP>:9443`).

Reply:
fixed